spec/rspec/core/example_spec.rb

```diff
@@ -344,11 +344,28 @@ def self.reliable_gc
         0 != GC.method(:start).arity # older Rubies don't give us options to ensure a full GC
       end
 
+      def expect_gc(opts)
+        get_all = opts.fetch :get_all
+
+        GC.disable
+        opts.fetch(:event).call
+        expect(get_all.call).to eq(opts.fetch :pre_gc)
+        GC.enable
+
+        20.times do
```

Comment:
Do you mind adding a comment here like:

> # see discussion on https://github.com/rspec/rspec-core/pull/1950
> # for why it's necessary to do this multiple times

It's odd enough that I think it warrants this :).